examples/observability/Makefile

```diff
@@ -0,0 +1,105 @@
+JAEGER_VERSION ?= v1.38.0
+JAEGER_NAMESPACE ?= observability
+PROM_NAMESPACE ?= monitoring
+LFC_NAMESPACE ?= keptn-lifecycle-controller-system
+PODTATO_NAMESPACE ?= podtato-kubectl
+
+.PHONY: install
+install:
+	@echo "-----------------------------------"
+	@echo "Create Namespace and install Jaeger"
+	@echo "-----------------------------------"
+	kubectl create namespace "$(JAEGER_NAMESPACE)" --dry-run=client -o yaml | kubectl apply -f -
```

Comment:
Using the manifest from the jaeger repository directly only supports using the `observability` namespace. For installing it into any other namespace, the manifest file needs to be modified first (replacing all `namespace: observability` occurences), as stated in their docs. Therefore it does not make sense to make this configurable via the `JAEGER_NAMESPACE` env var.

Reply:
fixed this